packages/ml-natural-language/lib/index.d.ts

```diff
@@ -241,17 +164,55 @@ export namespace MLKitLanguage {
     ): Promise<IdentifiedLanguage[]>;
 
     /**
-     * Returns a new instance of SmartReplyConversation.
+     * Returns suggested replies for a conversation.
      *
      * #### Example
      *
      * ```js
-     * const conversation = firebase.mlKitLanguage().newSmartReplyConversation();
+     * const replies = await firebase.mlKitLanguage().suggestReplies([
+     *   { text: 'We should catchup some time!', },
```

Comment:
```suggestion
     *   { text: "Hey, long time no speak!", },
```

Or w/e - you have 2 duplicate lines, doesn't make sense.